src/app/book/pages/book-list/book-list.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="buttons-container-admin" *ngIf="admin">
+  <button class="button">
+    <a class="add-book-link link" routerLink="/admin/product/add">Add new book</a>
+  </button>
+</div>
+<ul class="books-list">
+  <li class="book-container" *ngFor="let book of books | async" appFont [color]="'red'">
+    <app-book class="book" [book]="book">
+      <div class="buttons-container" *ngIf="!admin else editButton">
+        <button class="button">
```

Comment:
same goes here
btw: it can be just styled link

Reply:
fixed